provider/docker/builder_test.go

```diff
@@ -0,0 +1,69 @@
+package docker
+
+import (
+	docker "github.com/docker/engine-api/types"
+	"github.com/docker/engine-api/types/container"
+	"github.com/docker/engine-api/types/network"
+	"github.com/docker/go-connections/nat"
+)
+
+func containerJSON(ops ...func(*docker.ContainerJSON)) docker.ContainerJSON {
+	c := &docker.ContainerJSON{
```

Comment:
not sure, but between `c`, `ctnr` or `cont`, I went lazy 👼  (didn't want to use `container` because it would override the `container` package)